data/collections/IS2SITMOGR4-cog.json

```diff
@@ -26,7 +26,7 @@
     "license": "MIT",
     "description": "ICESat-2 L4 Monthly Gridded Sea Ice Thickness (COGs)",
     "stac_version": "1.0.0",
-    "dashboard:is_periodic": true,
```

Comment:
Does this remain `true` for MAAP?

Reply:
no idea. I might as well remove all the changes to the input files anyways since it's not relevant.

Reply:
simply reverted that change.